.github/workflows/continuous-deployment.yml

```diff
@@ -396,6 +396,54 @@ jobs:
           AZURE_KEY_VAULT_URL=${{ secrets.AZURE_KEY_VAULT_URL }} \
           PACER_TOKEN_SECRET_NAME=${{ secrets.PACER_TOKEN_SECRET_NAME }}"
 
+  integration-test:
+    defaults:
+      run:
+        working-directory: backend/functions
+
+    runs-on: ubuntu-latest
+    needs: [build-info]
```

Comment:
This line was modified temporarily to expedite execution of the integration test sooner. This should be reverted back to the following after validation of the job.
`needs: [build-info, deploy-webapp, deploy-service]`